### PR TITLE
Vv 71 trust manager flow add hospital code and status, removed URLs

### PR DESCRIPTION
**Pull request by nax2uk**
# What
  Added hospital code and hospital status to relevant hospital pages and removed support and survey URL from UI.

# Why
   Updated User requirements
# Screenshots
image
image

# Notes

### CODE REVIEW

File: pages/trust-admin/hospitals/add.js [hunk 1]
```diff
@@ -24,12 +24,13 @@ const AddAHospital = ({ error, trustId }) => {
...
   const getGenericError = () => {
     return {
-      id: 'generic-error',
+      id: "generic-error",
       message: "Something went wrong, please try again later.",
     };
   };
 
   const submit = async (payload) => {
+    console.log(payload);
```
Comment (faissalMT) [inline pages/trust-admin/hospitals/add.js:33]:
```suggestion
```

File: src/components/EditHospitalForm/index.js [hunk 2]
```diff
@@ -91,43 +87,48 @@ const EditHospitalForm = ({ errors, setErrors, hospital = {}, submit }) => {
...
+          <Select
+            id="hospital-status"
+            className="nhsuk-input--width-10 nhsuk-u-width-one-half"
+            prompt="Choose a hospital status"
+            options={[
+              { id: 1, name: "active" },
+              { id: 2, name: "disabled" },
+            ]}
+            onChange={(event) => {
+              setHospitalStatus(
```
Comment (faissalMT) [inline src/components/EditHospitalForm/index.js:122]: Consider moving this out into its own function

Comment (nax2uk) [reply src/components/EditHospitalForm/index.js]: I have created a SelectStatus component to handle hospital Status and ward Status (and other future statuses). Please review now. Thanks.

File: src/components/EditHospitalForm/index.js [hunk 3]
```diff
@@ -5,17 +5,15 @@ import Heading from "../Heading";
...
-  const [hospitalSurveyUrl, setHospitalSurveyUrl] = useState(
-    hospital.surveyUrl
-  );
-  const [hospitalSupportUrl, setHospitalSupportUrl] = useState(
-    hospital.supportUrl
-  );
+  const [hospitalStatus, setHospitalStatus] = useState(hospital.status);
+  const [hospitalCode, setHospitalCode] = useState(hospital.code);
+
+  const action = hospital.id ? "Edit" : "Add";
```
Comment (faissalMT) [inline src/components/EditHospitalForm/index.js:16]: It's best to avoid the ternary operator for things like this as it makes things unclear

Comment (nax2uk) [reply src/components/EditHospitalForm/index.js:16]: This was actually the previous code, which I re-added to the top of the code block. I think they wanted to combined two actions (Edit and Add) in one. How should I change the code? Should I add an if statement? Please suggest. Thanks.

Comment (faissalMT) [reply src/components/EditHospitalForm/index.js:16]: Yeah an `if` would be better

Comment (nax2uk) [reply src/components/EditHospitalForm/index.js]: If statement added. Please review, thank you.

**Review by faissalMT — APPROVED** [review]

**Pull request merged**